deploy/velero_cluster_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: velero
+subjects:
+- kind: ServiceAccount
+  name: velero
+  namespace: openshift-velero
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Are all the perms in cluster-admin required for velero? Is there any way to pair down exactly what permissions it needs and create a new cluster role with exactly those permissions?

Reply:
Unfortunately, yes. For backups, velero itself needs to read (GET/LIST/WATCH) every object in the cluster.. for restore, it needs to write (PUT/UPDATE/PATCH) every object in the cluster.